packages-next/keystone/src/___internal-do-not-use-will-break-in-patch/admin-ui/pages/ListPage/index.tsx

```diff
@@ -80,9 +80,11 @@ function useQueryParamsFromLocalStorage(listKey: string) {
   const storeableQueries = ['sortBy', 'fields'];
 
   const resetToDefaults = () => {
+    console.log('we are being called here', `keystone.list.${listKey}.list.page.info`);
+    console.log(localStorage.getItem(`keystone.list.${listKey}.list.page.info`));
     localStorage.removeItem(`keystone.list.${listKey}.list.page.info`);
     router.replace({
-      query: null,
+      pathname: router.pathname,
```

Comment:
Can you try this and see if it works?
```suggestion
      query: {},
```

Reply:
@mitchellhamilton it does not :(

Reply:
That's interesting, I'm guessing that's what caused not being able to remove a filter sometimes.